scribejava-apis/src/main/java/com/github/scribejava/apis/SalesforceApi.java

```diff
@@ -0,0 +1,72 @@
+package com.github.scribejava.apis;
+
+import java.security.KeyManagementException;
+import java.security.NoSuchAlgorithmException;
+
+import javax.net.ssl.HttpsURLConnection;
+import javax.net.ssl.SSLContext;
+
+import com.github.scribejava.apis.salesforce.SalesforceJsonTokenExtractor;
+
+/**
+ * This class is an implementation of the Salesforce OAuth2 API. 
+ */
+
+import com.github.scribejava.core.builder.api.DefaultApi20;
+import com.github.scribejava.core.extractors.TokenExtractor;
+import com.github.scribejava.core.model.OAuth2AccessToken;
+import com.github.scribejava.core.model.Verb;
+
+public class SalesforceApi extends DefaultApi20 {
+	
+	// Constants
+	private static final String BASE_URL = "https://login.salesforce.com/services/oauth2";
+	private static final String ACCESS_URL = BASE_URL + "/token?grant_type=authorization_code";
+	private static final String AUTHORIZE_URL = BASE_URL + "/authorize";
+
+    protected SalesforceApi() {
+    	SSLContext sc;
+		try {
+			String protocol = SSLContext.getDefault().getProtocol();
+			if (!protocol.equals("TLSv1.2") || !protocol.equals("TLSv1.1")) { // This is important!
+				sc = SSLContext.getInstance("TLSv1.2");
+				sc.init(null, null, null);
+				HttpsURLConnection.setDefaultSSLSocketFactory(sc.getSocketFactory());				
+			}
+		} catch (NoSuchAlgorithmException e) {
+			e.printStackTrace();
+		} 
+		catch (KeyManagementException e) {
+			e.printStackTrace();
```

Comment:
doesn't look like a good code.
We should notify users and let them know about this exceptions, aren't we?